irohad/ordering/impl/kick_out_proposal_creation_strategy.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_KICK_OUT_PROPOSAL_CREATION_STRATEGY_HPP
+#define IROHA_KICK_OUT_PROPOSAL_CREATION_STRATEGY_HPP
```

Comment:
IROHA_SUPER_KICK_OUT_MEGA_PROPOSAL_ULTRA_CREATION_STRATEGY_9000_HPP
:D

Reply:
Could you suggest a more appropriate name?

Reply:
No, It was not an issue